README.md

```diff
@@ -104,17 +104,17 @@ The simplest syntax to use is the inline syntax. This syntax resembles
 Scala's use of underscores to define anonymous functions like `_ + _`.
 
 Since underscore is used for existential types in Scala (and it is
-probably too late to change this syntax), we use `?` for the same
-purpose. We also use `+?` and `-?` to handle covariant and
+probably too late to change this syntax), we use `*` for the same
```

Comment:
The parenthetical is now incorrect, and should probably be replaced by a link to http://dotty.epfl.ch/docs/reference/changed-features/wildcards.html